module2/validation.js

```diff
@@ -0,0 +1,13 @@
+import Joi from 'joi';
+
+const schema = Joi.object().keys({
+  login: Joi.string().alphanum().min(1).required(),
+  password: Joi.string().alphanum().min(6).required(),

```

Comment:
This password validation is wrong. I am able to create password which will contain only letters.

Reply:
Fixed